migrations/1658554082950_add-linkedin-field.sql

```diff
@@ -0,0 +1,7 @@
+-- Up Migration
```

Comment:
if I do a full clean-up of my DB than start the app, none if those migrations are performed, did I missed something to do ? Bellow the migrations done by the app on my local after a full clean-up

1	"1638806290414_init-db"	"2023-04-13 20:58:00.571968"
2	"1638806616344_create-users-keycloak-id-index"	"2023-04-13 20:58:00.571968"
3	"1641590666695_add-more-user-data-column"	"2023-04-13 20:58:00.571968"
4	"1643221616684_add-config-column"	"2023-04-13 20:58:00.571968"
5	"1643307922961_add-saved-filters"	"2023-04-13 20:58:00.571968"
6	"1643735654495_add-default-saved-filter-column"	"2023-04-13 20:58:00.571968"
7	"1646761533686_add-user-field-commercial-user-reason"	"2023-04-13 20:58:00.571968"
8	"1646927694320_add-email"	"2023-04-13 20:58:00.571968"
9	"1648215821881_add-user-sets"	"2023-04-13 20:58:00.571968"
10	"1648740300750_modify-user-sets"	"2023-04-13 20:58:00.571968"
11	"1648840444303_update-usersets-put-all-in-content"	"2023-04-13 20:58:00.571968"
12	"1676914790527_update-research-area-to-be-a-list"	"2023-04-13 20:58:00.571968"

Reply:
Nevermind `docker-compose up --build` did the trick

Reply:
Got something interessting, it's not possible to migrate from the version on main/master to that one because of the following:
```
Error: Not run migration 1658554082950_add-linkedin-field is preceding already run migration 1676914790527_update-research-area-to-be-a-list
users-api-app-1  |     at checkOrder (/app/node_modules/node-pg-migrate/dist/runner.js:110:19)
users-api-app-1  |     at exports.default (/app/node_modules/node-pg-migrate/dist/runner.js:156:13)
```
I think all the news migrations should have a name prefixed with a greater timstamp than `1676914790527`